pkg/ds/daemon_set.go

```diff
@@ -170,6 +175,15 @@ func (ds *daemonSet) EligibleNodes() ([]types.NodeName, error) {
 	return ds.scheduler.EligibleNodes(ds.Manifest, ds.NodeSelector)
 }
 
+func (ds *daemonSet) metricName(name string) string {
+	return strings.Join([]string{
+		"daemonset",
+		ds.PodID().String(),
```

Comment:
for convenience, show pod ID. Maybe could also show ClusterName? I'd have to add a String() to ClusterName.

If we show both pod ID and cluster name, do we need to show the DS ID? Only if people make daemon sets that clash on both these properties...